.github/workflows/test.yml

```diff
@@ -13,7 +13,6 @@ jobs:
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
         include:
           - python-version: "3.10"
-            os: ubuntu-20.04
```

Comment:
Sadly it doesn't, which is why some folks recommend `codecov` instead. See e.g. lemurheavy/coveralls-public#613, which started in 2015 and is still going strong in 2023.

Reply:
> See e.g. https://github.com/lemurheavy/coveralls-public/issues/613, which started in 2015 and is still going strong in 2023.

That seems to be related to using different CI systems though. In theory I think it's possible to do the merge within Github Actions. On the other hand, I think my suggestion of having tests that use mocking to force use of the fallback implementation will make it possible for Linux to have a superset of MacOS coverage, in which case I don't think it's necessary to run coverage on MacOS at all.

> By the way, I just reverted it to the way it was before https://github.com/ska-sa/aiokatcp/pull/70.

Not exactly the way it was, because before #70 the tests weren't run on MacOS at all.

Reply:
> Not exactly the way it was...

Right, forgot about that part :-)

As I discovered after you replied (blame GitHub flakiness :-)), it looks like the merge does happen. I'll try that route first.

Reply:
It does. The latest results claim that `Coverage decreased (-0.1%) to 93.406%`. If you drill down to the details, you'd see two jobs (linux + macOS) and a merged total of 93.41%:

<img width="1142" alt="Screenshot 2023-09-11 at 11 56 34" src="https://github.com/ska-sa/aiokatcp/assets/392144/a3ef2e1c-8f8d-4184-8f5f-e2a033a656f7">

Each job has poor coverage on `adjtimex.py` but the combined coverage is 95.52%.

Reply:
Nice!